src/domain/newsletter/__tests__/SubscribeNewsletterPage.test.tsx

```diff
@@ -109,10 +105,10 @@ describe('SubscribeNewsletterPage', () => {
 
     await waitFor(() => {
       expect(
-        screen.getByRole('heading', {
-          name: /valitettavasti tilauksen lähettäminen epäonnistui! yritäthän myöhemmin uudelleen\./i,
-        })
-      ).toBeInTheDocument();
+        screen.getAllByText(
```

Comment:
This should be reverted. I tested with the old implementation and it's working ok

Reply:
actually, I tried with `GetByText`